context/ethereumContext.tsx

```diff
@@ -79,6 +79,7 @@ type ContextProps = {
   isExecuting: boolean
   executionState: IExecutionState
   vmError: string | undefined
+  isForksLoaded: boolean
```

Comment:
Probably plural wording?

```suggestion
  areForksLoaded: boolean
```